src/core/Global.ttslua

```diff
@@ -297,6 +297,39 @@ function onPlayerAction(player, action, targets)
       trash.putObject(target)
     end
     return false
+  elseif action == Player.Action.PickUp then
+    local pickedCards = {}
+    for _, target in ipairs(targets) do
+      if target.type == "Card" then
+        table.insert(pickedCards, target)
+      end
+    end
+    if #pickedCards < 6 then
+      for _, pickedCard in ipairs(pickedCards) do
+        local hitList = Physics.cast({
+          origin = pickedCard.getPosition(),
+          direction = {0,1,0},
+          type = 3,
+          size = Vector(2.4, 0, 3.4),
+          orientation = pickedCard.getRotation(),
+          max_distance = 0.5,
+        })
+        for _, token in ipairs(hitList) do
+          local pickedToken = token.hit_object
+          if pickedToken.type == "Tile" then
+            pickedCard.addAttachment(pickedToken)
+          end
+        end
+        Wait.condition(
+          function()
+            pickedCard.removeAttachments()
```

Comment:
This needs nil handling since the card could enter a container:
```suggestion
            if pickedCard ~= nil then
              pickedCard.removeAttachments()
            end
```